# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../amarok/mysql.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur amarok-2.0.1.1-orig/cmake/modules/FindMySQLAmarok.cmake amarok-2.0.1.1/cmake/modules/FindMySQLAmarok.cmake
--- amarok-2.0.1.1-orig/cmake/modules/FindMySQLAmarok.cmake	2009-01-22 21:42:25.000000000 +0000
+++ amarok-2.0.1.1/cmake/modules/FindMySQLAmarok.cmake	2009-01-22 21:44:36.000000000 +0000
@@ -16,7 +16,7 @@
 find_path(MYSQL_INCLUDE_DIR mysql.h
    ~/usr/include/mysql
    /opt/local/include/mysql5/mysql
-   /opt/mysqle/include/mysql
+   /opt/mysql/include/mysql
    /usr/include/mysql
    /usr/local/include/mysql
    /opt/local/include/mysql
@@ -26,7 +26,7 @@
    PATHS
    ~/usr/lib/mysql
    /opt/local/lib/mysql5/mysql
-   /opt/mysqle/lib/mysql
+   /opt/mysql/lib/mysql
    /usr/lib/mysql
    /usr/lib64/mysql
    /usr/lib64
@@ -38,7 +38,7 @@
    PATHS
    ~/usr/lib/mysql
    /opt/local/lib/mysql5/mysql
-   /opt/mysqle/lib/mysql
+   /opt/mysql/lib/mysql
    /usr/lib/mysql
    /usr/lib64/mysql
    /usr/local/lib/mysql
